tests/runtests.pl

```diff
@@ -15,17 +15,20 @@
 my $output = `id`;
 $output =~ /uid=\d+\((\w+)\).*/;
 my $unix_user = $1;
-$output =~ /context=(\w+):(\w+):(\w+)/;
+$output =~ /context=(\w+):(\w+):(\w+):(\w+)-([\w:\.]+)/;
 my $selinux_user = $1;
 my $selinux_role = $2;
 my $selinux_type = $3;
+my $selinux_sen = $4;
```

Comment:
Can you provide or point to some background to this, since that's the first I've seen "eff" relating to selinux.  The code in mls_sid_to_context() uses sens(sensitivity?), cats(categories?), levels.  Other labels that might work?  "mls", "range"(though this might replace "clr")?  I don't remember the exact source I used to come up with those variable names originally, but from what I recall, there seemed to be a well-established usage in that context of user/role/type/sen/clr.